index.html

```diff
@@ -128,6 +128,11 @@
 <pre><code>./run-tests.sh</code></pre>
 
 <h2>
+<a name="api-sparkr" class"anchor" href="#api-sparkr"><span class="octicon octicon-link"></span></a>SparkR package API documentation</h2>
+
+<a href="rdocs/00Index.html">Documentation for package ‘SparkR’</a>
```

Comment:
Ah I actually meant that knitr generates a `index.html` which has two frames. It is already a part of the PR https://github.com/amplab-extras/SparkR-pkg/pull/227/files#diff-39721893066db73eb1158d7561a111c1 as you can see